### PR TITLE
feat: Activate ubuntu_lints

### PR DESCRIPTION
**Pull request by spydon**
Accidentally the explicit lint rules in `ubuntu_lints` weren't activated before since they were missing `-` in front of them.

This PR activates all the rules and fixes all occurrences within the repository (and adds exceptions to tests/examples with prints etc).

### CODE REVIEW

File: packages/platform_linux/pubspec.yaml [hunk 1]
```diff
@@ -14,6 +14,6 @@ dependencies:
   platform: ^3.1.0
 
 dev_dependencies:
-  ubuntu_lints: ^0.1.0
   mockito: ^5.4.2
   test: ^1.21.0
+  ubuntu_lints: ^0.1.0
```
Comment (d-loose) [inline packages/platform_linux/pubspec.yaml:19]: Just double-checking: `melos bootstrap` adds its own overrides to use the local package, right? So this version needs to be bumped manually once 0.2.0 gets released?

File: packages/ubuntu_widgets/lib/src/log_view.dart [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: always_put_required_named_parameters_first
```
[the remark this answers is not in the document]
Comment (d-loose) [reply packages/ubuntu_widgets/lib/src/log_view.dart]: There are a couple more

File: packages/ubuntu_widgets/lib/src/override_mouse_cursor.dart [hunk 3]
```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: always_put_required_named_parameters_first
```
Comment (d-loose) [inline packages/ubuntu_widgets/lib/src/override_mouse_cursor.dart:1]: here

File: packages/ubuntu_widgets/lib/src/slide_show.dart [hunk 4]
```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: always_put_required_named_parameters_first
```
Comment (d-loose) [inline packages/ubuntu_widgets/lib/src/slide_show.dart:1]: here

File: packages/ubuntu_widgets/lib/src/log_view.dart (hunk 2 above)
[the remark this answers is not in the document]
Comment (spydon) [reply packages/ubuntu_widgets/lib/src/log_view.dart]: Fixed them all now, there were some of other types that could be removed too.

**Pull request merged**